src/Arcus.Messaging.Tests.Runtimes.AzureFunction.EventHubs.InProcess/OrderFunction.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.EventHubs;
+using Arcus.Messaging.Abstractions.EventHubs.MessageHandling;
+using Arcus.Messaging.AzureFunctions.EventHubs;
+using Azure.Messaging.EventHubs;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Messaging.Tests.Runtimes.AzureFunction.EventHubs.InProcess
+{
+    public class OrderFunction
```

Comment:
Orders is maybe not a good example, as you'll typically not use EventHubs for things like Orders, etc...   An Order should be processed exactly once, and that is not guaranteed with EventHubs :)

Reply:
Yes, but this is for the Docker integration tests, remember? We decided that the extra work to use another type is not worthwhile for the benefit it brings.